metadata-ingestion/src/datahub/cli/specific/structuredproperties_cli.py

```diff
@@ -61,3 +63,85 @@ def get(urn: str, to_file: str) -> None:
                 )
         else:
             click.secho(f"Structured property {urn} does not exist")
+
+
+@properties.command(
+    name="list",
+)
+@click.option("--details/--no-details", is_flag=True, default=True)
+@click.option("--to-file", required=False, type=str)
```

Comment:
imo having the to file option is strange - it should just print to stdout, and the user can redirect the output to a file if they want

Reply:
our structured properties file option actually supports multiple structured properties in the same file ... so it makes sense to have a --to-file option which writes out in the same yaml format that we support on the `upsert` call.